tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -235,7 +286,18 @@ def parameters(cls):
                 description="The list of output layers names (optional),"
                             " if specified there should be exactly 3 output layers provided."
             ),
-            'anchor_masks': ListField(optional=True, description='per layer used anchors mask')
+            'anchor_masks': ListField(optional=True, description='per layer used anchors mask'),
+            'cells': ListField(
+                optional=True, default=[13, 26, 52],
```

Comment:
please note that usage this values as default will require update at least config for mobilefacedet-v1-mxnet since it has different cell size. Also not sure about model yolo-v3-spp (with 608x608 input resolution)

Reply:
I solved this problem, may be little bit messy, but should work for both models, if they keep 4d shape. Also original config for yolo-v3-tf should work with flat output.

Reply:
what about check `mobilefacedet-v1-mxnet`? This model also use this adapter and have different from standart cells

Reply:
Well, the idea is to use `cells` option only if output not 3d or `do_reshape=True`. The `mobilefacedet-v1-mxnet` has 3d output tensor, so default cells will be ignored. This is not good idea, more silver bullet, but it is preferable than new adapter for now.

Reply:
ok, now I am patient that we do not break something :)